include/triton/Dialect/TritonGPU/Transforms/Schedule.h

```diff
@@ -129,13 +129,13 @@ class CoarseSchedule {
   Cluster splitClusterBefore(Operation *op, scf::ForOp forOp);
 
   // Check if op a will show up before op b in the final unrolled code.
-  bool isOpBefore(Operation *a, Operation *b) const;
+  bool isOpBefore(Operation *a, Operation *b);
```

Comment:
nit: would it be cleaner to mark `opToStageAndCluster` as mutable instead? not a strong opinion maybe @pawelszczerbuk has a preference

Reply:
actually you don't even need to drop const (i just need to brush up on my LLVM data structures.........)